qa/suites/rbd/iscsi/cluster/fixed-3.yaml

```diff
@@ -0,0 +1,4 @@
+roles:
+- [mon.a, osd.0, osd.1, osd.2, gateway.0]
+- [mon.b, osd.3, osd.4, osd.5, gateway.1]
+- [mon.c, mgr.x, osd.6, osd.7, gateway.2]
```

Comment:
You can probably just use 2 gateways so that the third node can run the iSCSI initiator

Reply:
Okay, will do that.